llvm/lib/CodeGen/GlobalISel/IRTranslator.cpp

```diff
@@ -2972,7 +2972,22 @@ bool IRTranslator::translateInsertElement(const User &U,
   Register Res = getOrCreateVReg(U);
   Register Val = getOrCreateVReg(*U.getOperand(0));
   Register Elt = getOrCreateVReg(*U.getOperand(1));
-  Register Idx = getOrCreateVReg(*U.getOperand(2));
+  const auto &TLI = *MF->getSubtarget().getTargetLowering();
```

Comment:
the TargetLowering should probably be an IRTranslator member if it isn't already